src/main/scala/cognite/spark/v1/CdfRelation.scala

```diff
@@ -12,21 +12,19 @@ abstract class CdfRelation(config: RelationConfig, shortNameStr: String)
     extends BaseRelation
     with Serializable {
   protected val shortName: String = shortNameStr
-  @transient lazy protected val itemsRead: Counter =
-    MetricsSource.getOrCreateCounter(config.metricsPrefix, s"$shortName.read")
-  @transient lazy protected val itemsCreated: Counter =
-    MetricsSource.getOrCreateCounter(config.metricsPrefix, s"$shortName.created")
-  @transient lazy protected val itemsUpdated: Counter =
-    MetricsSource.getOrCreateCounter(config.metricsPrefix, s"$shortName.updated")
-  @transient lazy protected val itemsDeleted: Counter =
-    MetricsSource.getOrCreateCounter(config.metricsPrefix, s"$shortName.deleted")
+  protected def itemsRead: Counter = getOrCreateCounter("read")
+  protected def itemsCreated: Counter = getOrCreateCounter("created")
+  protected def itemsUpdated: Counter = getOrCreateCounter("updated")
+  protected def itemsDeleted: Counter = getOrCreateCounter("deleted")
```

Comment:
```suggestion
  @transient lazy protected val itemsRead: Counter = getOrCreateCounter("read")
  @transient lazy protected val itemsCreated: Counter = getOrCreateCounter("created")
  @transient lazy protected val itemsUpdated: Counter = getOrCreateCounter("updated")
  @transient lazy protected val itemsDeleted: Counter = getOrCreateCounter("deleted")
```
we can still keep it being a field?